tools/standalone-binary-distro/make_distro_builds.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/zsh
+
+gh auth status;
+if [[ $? != 0 ]]; then
+  exit 1
+fi
+
+SDK_VERSION=$(git describe --exact-match --tags)
+if [[ $? != 0 ]]; then
+  echo "❌ Aborting: HEAD doesn't have a tag!"  
+  exit 2
+elif [[ `git status --porcelain` ]]; then
+  echo "❌ Aborting: Working directory has changes!"
+  exit 3
+elif [ ! -f "Cartfile" ]; then
+	echo "❌ Aborting: \`make_distro_builds.sh\` must be run from the same place with `Cartfile`"
+  exit 4
+fi
+OUT_FILENAME="Datadog-$SDK_VERSION.zip"
+
+# create temporary xcconfig for carthage build
+set -euo pipefail
+xcconfig=$(mktemp /tmp/static.xcconfig.XXXXXX)
+trap 'rm -f "$xcconfig" ;' INT TERM HUP EXIT
+echo "BUILD_LIBRARY_FOR_DISTRIBUTION = YES" >> $xcconfig
+export XCODE_XCCONFIG_FILE="$xcconfig"
+
+# clear existing carthage artifacts
+rm -rf Carthage/
+rm Cartfile.resolved
```

Comment:
Why deleting [`Cartfile.resolved`](https://github.com/DataDog/dd-sdk-ios/blob/dogfooding/Cartfile.resolved)? If removed the `carthage bootstrap` will use up-to-date versions of our dependencies, not the one that we work against.

Reply:
this may make `carthage bootstrap` unneeded, unresolving the discussion for a moment ⏳